vector/apps/web/lib/testing/runner/agent-executor.ts

```
@@ -0,0 +1,515 @@
+  private async applyObjectProposal(proposal: ObjectProposal, toolCalls: ToolCall[]): Promise<void> {
+    this.log(`🔨 Object Operations: ${proposal.id}`);
+    this.log(`   Operations: ${proposal.ops.length}`);
+
+    for (const op of proposal.ops) {
+      const startTime = Date.now();
+
+      switch (op.op) {
+        case 'create_instance':
+          this.log(`   ➕ Create: ${op.className} at ${op.parentPath}`);
+          const name = (op.props as any)?.Name || op.className;
+          this.env.createInstance(op.parentPath!, op.className!, name, op.props);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)